RobotEV3/src/main/resources/ev3c4ev3.properties

```diff
@@ -1,6 +1,13 @@
 robot.plugin.group = ev3
 robot.plugin.factory = de.fhg.iais.roberta.factory.EV3Factory
 robot.plugin.compiler.resources.dir = RobotEV3/crossCompilerResources/
+
+robot.plugin.compiler.executable.linux = arm-linux-gnueabi-g++
+robot.plugin.compiler.executable.arm = arm-c4ev3-linux-uclibceabi-g++
+robot.plugin.compiler.executable.mac = arm-none-linux-gnueabi-g++
+robot.plugin.compiler.executable.windows = ""
```

Comment:
Best leave windows out if it does not define anything, that way the default will be used (although that is currently empty as well)